ios-trivia-game/ViewControllers/GameOptionsViewController.swift

```diff
@@ -78,18 +84,32 @@ class GameOptionsViewController: UIViewController {
 
     // Make an api call to create a game room, and then go to CountdownTimerViewController if it succeeds.
     @IBAction func onStartGameClicked(_ sender: Any) {
-        let newGame = GameRoom(id: FirebaseClient.instance.createGameRoomId().key, name: self.nameOfGameroom, currentNumPlayers: 0, maxNumPlayers: self.numOfPlayers, state: GameRoom.State.idle, isPublic: self.isPublic, currentQuestion: 0, maxNumQuestions: self.numOfQuestions)
-
-        FirebaseClient.instance.createGame(gameRoom: newGame.getJson() as NSDictionary, complete: {_, ref in
-            let roomId = ref.key
+        FirebaseClient.instance.getRandomQuestions(categoryId: category!, maxNumOfQuestions: numOfQuestions!, complete: {(questionList) in
+            let newGame = ["max_num_of_questions" : self.numOfQuestions ?? 0,
```

Comment:
why did you wipe out line 81?
You're duplicating the default values in the initializer for creating a game

revert this